app/api/margatsni/v1/users/users.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Margatsni
+  module V1
+    module Users
+      class Users < Margatsni::V1::BaseV1
+        helpers do
+          def represent_user_with_token(user)
+            present user, with: Margatsni::V1::Entities::User
```

Comment:
As I mentioned before, it would be better to not add any kind of tokens to user entity. But you can present user with the token like below:
```
present { token: token }
present user, with: Margatsni::V1::Entities::User
```
Please note I wrote pseudocode and it probably won't work, because you'll need to pass user with some key to merge multiple presents.